server/readme.md

```diff
@@ -1,16 +1,16 @@
 # Flask (sqlite) Book Server
 
-Starting off, run the following commands in your terminal while in the server folder for the initial setup:
+Starting off, run the following commands in your terminal while in the `server` folder for the initial setup:
 
-```js
-pipenv shell
-pipenv install
+```
+$ pipenv shell
+$ pipenv install
 ```
 
```

Comment:
I think we need one more change to this file.  We need to inform the user on how to create the `database`